.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,24 @@
+name: Spell Checking
+
+on: [push, pull_request]
+
+jobs:
+  codespell:
+    name: Check spelling with codespell
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
Any reason to use Python 3.8? It appears to be scheduled for end-of-life next month.